packages/utils/src/bigNumber.ts

```diff
@@ -0,0 +1,9 @@
+import BN from 'bignumber.js';
+
+export const BigNumber = BN.clone({
```

Comment:
I am not sure if I like BN in utils package, because it was meant to be zero dependency. But probably fine.

Reply:
I wanted to avoid making a new package just for this, so utils seemed like the best place for it.

Reply:
hmm, thats right, I sort of forgot about this zero-dependency rule when I advised @martykan to do it like this. Maybe an option would be to list this bignumber as a peer dep, but it is probably even more confusing

Reply:
Seems like it's possible to mark the peer dep as optional, I think that could work